src/routes/about/+page.svelte

```diff
@@ -1,21 +1,27 @@
 <script>
-  import Button from "$lib/components/Button.svelte";
+	import Button from "$lib/components/Button.svelte";
 </script>
 
 <main class="prose p-4">
-  <h1>
-    <Button href="/">Seirea</Button>
-  </h1>
-  <h2>What is Seirea?</h2>
-  <p>
-    Seirea is an open source grade viewer. You can view the grades of all your
-    classes and assignments.
-  </p>
+	<h1>
+		<Button href="/">Seirea</Button>
+	</h1>
+	<h2>What is Seirea?</h2>
+	<p>
+		Seirea is a grade viewer. You can view the grades of all your classes and
+		assignments.
+	</p>
 
-  <p>
-    Seirea is created by
-    <span class="font-bold">Stanley Wang</span>
-    and
-    <span class="font-bold">Yavor Kolev</span>.
-  </p>
+	<p class="bg-red-400 rounded-md p-2 font-bold">
+		Seirea is still in development! Expect bugs, changes, and fixes. Please
+		report issues on the GitHub.
+	</p>
+
+	<p>
+		Seirea is open-source software created by
+		<span class="font-bold">Stanley Wang</span>
+		and
+		<span class="font-bold">Yavor Kolev</span>. Contribute at
```

Comment:
You could also mention @paranthropusgoober

Reply:
He does not want his real name associated with this project.

Reply:
Alright then